packages/pn-pa-webapp/src/pages/components/NewNotification/PreliminaryInformations.tsx

```diff
@@ -117,6 +119,12 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
     fetchGroups();
   }, [fetchGroups]);
 
+  const InfoTooltip = ({ tooltip }: { tooltip: string | ReactNode }) => (
```

Comment:
I suggest to  embed the `Tooltip` inside a `InputAdornment position='end'` element. Besides being more aligned with MUI documentation, in this way the `InfoOutlined` icon appears with color #5C6F82 as indicated in the Figma document
![image](https://github.com/pagopa/pn-frontend/assets/5519535/94655a7c-6107-4fa5-ab80-fefda9a67464)
Note that if `InputAdornment` is not used, then the color changes to #17324D, the one used for texts.